device/bgp/bgpgracefulrestart.yaml

```diff
@@ -0,0 +1,50 @@
+components:
+  schemas:
+    Bgp.GracefulRestart:
+      description: >-
+        The Graceful Restart Capability (RFC 4724) is a BGP capability that can be
+        used by a BGP speaker to indicate its ability to preserve its forwarding
+        state during BGP restart.
+        The Graceful Restart (GR) capability is advertised in OPEN messages sent between BGP peers.
+        After a BGP session has been established, and the initial routing update has been completed, 
+        an End-of-RIB (Routing Information Base) marker is sent in an UPDATE message to convey information 
+        about routing convergence.
+      type: object
+      properties:
+        enable_gr:
+          description: >-
+            If enabled, Graceful Restart capability is advertised in BGP OPEN messages.
+          type: boolean
+          default: false
+          x-field-uid: 1
+        restart_time:
+          description: >-
+            This is the estimated time (in seconds) it will take for the
+            BGP session to be re-established after a restart.  This can be
+            used to speed up routing convergence by its peer in case
+            the BGP speaker does not come back after a restart.
+          type: integer
+          format: uint16
```

Comment:
As per Ashutosh , we cannot have uint16 so we should use uint32 instead  probably. Need to check all fields where fmt has been used. Also, this model should be merged with main only after otgen supports new fmts uint32 and uint64 used in this PR

Reply:
done